lightning/src/ln/channel.rs

```diff
@@ -5455,13 +5458,12 @@ impl<'a, Signer: Sign, K: Deref> ReadableArgs<(&'a K, u32)> for Channel<Signer>
 			config.as_mut().unwrap().forwarding_fee_proportional_millionths = Readable::read(reader)?;
 			config.as_mut().unwrap().cltv_expiry_delta = Readable::read(reader)?;
 			config.as_mut().unwrap().announced_channel = Readable::read(reader)?;
-			config.as_mut().unwrap().commit_upfront_shutdown_pubkey = Readable::read(reader)?;
+			// config.as_mut().unwrap().commit_upfront_shutdown_pubkey = Readable::read(reader)?;
```

Comment:
Note that because the type of `commit_upfront_shutdown_pubkey` has changed from a bool (represented as a single byte) to an `Option`, we have to change how we read it here - otherwise we try to read a byte indicating if the value is `Some` or `None` and then try to read a second byte for the value.